docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md

```diff
@@ -0,0 +1,453 @@
+    | ----------------------------------------------------- | ------------------------------ | ------------------- |
+    |       `mx.nd.flatten(*args, **kwargs)`                |            `mx.npx.batch_flatten(*args, **kwargs)`                    |                moved to `npx` namespace with new name `batch_flatten`            |
+    |       `mx.nd.concat(a, b, c)`                |            `mx.np.concatenate([a, b, c])`                    |              - moved to `np` namespace with new name `concatenate`. <br> - use list of ndarrays as input rather than positional ndarrays           |
+    |        `mx.nd.stack(a, b, c)`                 |            `mx.np.stack([a, b, c])`                    |              - moved to `np` namespace. <br> - use list of ndarrays as input rather than positional ndarrays           |
+    |      `mx.nd.SliceChannel(*args, **kwargs)`              |            `mx.npx.slice_channel(*args, **kwargs)`                 |              - moved to `npx` namespace with new name `slice_channel`.          |
+    |      `mx.nd.FullyConnected(*args, **kwargs)`              |            `mx.npx.fully_connected(*args, **kwargs)`                 |              - moved to `npx` namespace with new name `fully_connected`.          |
+    |      `mx.nd.Activation(*args, **kwargs)`              |            `mx.npx.activation(*args, **kwargs)`                 |              - moved to `npx` namespace with new name `activation`.          |
+    |      `mx.nd.Activation(*args, **kwargs)`              |            `mx.npx.activation(*args, **kwargs)`                 |              - moved to `npx` namespace with new name `activation`.          |
+    |      `mx.nd.elemwise_add(a, b)`              |            `a + b`                 |              - Just use ndarray python operator.          |
+    |      `mx.nd.elemwise_mul(a, b)`              |            `mx.np.multiply(a, b)`                 |              - Use `multiply` operator in `np` namespace.          |
+
+2. Operators migration with multiple steps: `mx.nd.mean` -> `mx.np.mean`:
+```{.python}
+import mxnet as mx
+# Legacy: calculate mean value with reduction on axis 1
+#         with `exclude` option on 
+nd_mean = mx.nd.mean(data, axis=1, exclude=1)
+
+# Numpy: no exclude option to users, but user can perform steps as follow
+axes = list(range(data.ndim))
+del axes[1]
+np_mean = mx.np.mean(data, axis=axes)
+```
+
+3. Random Operators
+    |                   Legacy Operators               |    NumPy Operators Equivalent    |   Changes  |
+    | ----------------------------------------------------- | ------------------------------ | ---------------------------- |
+    |       `mx.random.uniform(-1.0, 1.0, shape=(2, 3))` <br> `mx.nd.random.uniform(-1.0, 1.0, shape=(2, 3))`                |            `mx.np.random.uniform(-1.0, 1.0, size=(2, 3))`                    |                For all the NumPy random operators, use **size** key word instead of **shape**           |
```

Comment:
```suggestion
    |       `mx.random.uniform(-1.0, 1.0, shape=(2, 3))` <br> `mx.nd.random.uniform(-1.0, 1.0, shape=(2, 3))`                |            `mx.np.random.uniform(-1.0, 1.0, size=(2, 3))`                    |                For all the NumPy random operators, use **size** keyword instead of **shape**           |
```